modules/price/src/Element/Price.php

```diff
@@ -27,16 +27,16 @@
  *
  * @FormElement("commerce_price")
  */
-class Price extends Textfield {
+class Price extends FormElement {
```

Comment:
@bojanz I looked at `\Drupal\Core\Render\Element\Textfield` and we weren't gaining anything out of extending it.  We weren't using any of the parent class's process or pre-render methods.